site/en/codelabs/openthread-hardware/index.lab.md

```diff
@@ -282,7 +282,7 @@ output and confirm that it is running:
 ```console
 $ cd ~/src/openthread
 $ sudo ./build/posix/src/posix/ot-daemon -v \
-    'spinel+hdlc+uart:///dev/ttyACM0?uart-baudrate=115200'
+    'spinel+hdlc+uart:///dev/ttyACM0?uart-baudrate=460800'
```

Comment:
Do you also need to update the baud rate mentioned further down in "Verify build"?

Reply:
When using native USB, there is no need to specify the baud rate. So I just removed that from the other commands below.